### PR TITLE
Address RoslynAnalysis bugs

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Also fixed an issue with FileBugs.yaml

### CODE REVIEW

File: tests/InstrEngineTests/InstrEngineTests/TargetAppCompiler.cs [hunk 1]
```diff
@@ -24,7 +24,11 @@ internal class TargetAppCompiler
     {
         private const string SourceExtension = "cs";
         private const string DynamicallyLinkedLibraryExtension = "dll";
+
+#pragma warning disable CA1823 //Avoid unused private fields
```
Comment (wiktork) [inline tests/InstrEngineTests/InstrEngineTests/TargetAppCompiler.cs:28]: Did you want to condition this field on !NETCOREAPP instead?

**Review by delmyers — APPROVED** [review]
:shipit:

**Pull request merged**